client/payment-details/dispute-details/dispute-due-by-date.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { dateI18n } from '@wordpress/date';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import classNames from 'classnames';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import type { EvidenceDetails } from 'wcpay/types/disputes';
+
+const DisputeDueByDate: React.FC< {
+	dueBy: EvidenceDetails[ 'due_by' ];
+} > = ( { dueBy } ) => {
+	const daysRemaining = Math.floor(
+		moment.unix( dueBy ).diff( moment(), 'days', true )
+	);
+	const respondByDate = dateI18n(
+		'M j, Y, g:ia',
+		moment( dueBy * 1000 ).toISOString()
+	);
+	return (
+		<span className="dispute-steps__steps__response-date">
+			{ respondByDate }
+			<span
+				className={ classNames( {
+					'dispute-steps__steps__response-date--urgent':
+						daysRemaining < 3,
+					'dispute-steps__steps__response-date--warning':
+						daysRemaining < 7 && daysRemaining > 2,
+				} ) }
+			>
+				{ daysRemaining > 0 &&
+					sprintf(
+						// Translators: %s is the number of days left to respond to the dispute.
+						_n(
+							'(%s day left to respond)',
+							'(%s days left to respond)',
+							daysRemaining,
```

Comment:
Is `daysRemaining` a string or numeric type?

Reply:
Nice catch. Changed to numeric placeholder in a83eebbd3de188c3c6680eabf9431015011fcf66.